Sprint-1/interpret/percentage-change.js

```diff
@@ -12,11 +12,17 @@ console.log(`The percentage change is ${percentageChange}`);
 // Read the code and then answer the questions below
 
 // a) How many function calls are there in this file? Write down all the lines where a function call is made
+// ***Answer***: There are 3 function calls in this file. The first is Number() and second is console.log. Line 4, line 5 and line 10.
 
 // b) Run the code and identify the line where the error is coming from - why is this error occurring? How can you fix this problem?
+// ***Answer***: replaceAll() is not a function. We can fix this problem by using replace() with a regular expression instead.
```

Comment:
I believe replace All is a function. Kindly check and verify.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replaceAll